### PR TITLE
record bloat

### PR DESCRIPTION
**Pull request by rminnich**
We don't even have to commit this if we don't want.

### CODE REVIEW

File: cmds/installcommand/installcommand.go [hunk 1]
```diff
@@ -147,6 +147,7 @@ func main() {
 	env := golang.Default()
 	env.Context.GOROOT = r("/go")
 	env.Context.GOPATH = r("/")
+	env.Context.CgoEnabled = false
```
Comment (rjoleary) [inline cmds/installcommand/installcommand.go]: Can you move this to a separate PR?

File: templates.go [hunk 2]
```diff
@@ -186,4 +186,99 @@ var templates = map[string][]string{
...
+	// ./u-root -build=bb bloat
+	//
+	// go version go1.10 linux/amd64
+	// -rwxr-xr-x 1 rminnich rminnich 11402356 Feb  7 11:07 /tmp/initramfs.linux_amd64.cpio
+	// go version go1.11.5 linux/amd64
+	// -rwxr-xr-x 1 rminnich rminnich 13021172 Feb  7 11:12 /tmp/initramfs.linux_amd64.cpio
+	// go version devel +4b3f04c63b Thu Jan 10 18:15:48 2019 +0000 linux/amd64
+	// -rwxr-xr-x 1 rminnich rminnich 13791444 Feb  7 11:15 /tmp/initramfs.linux_amd64.cpio
+	// The trend is not good.
+	"bloat": {
```
Comment (rjoleary) [inline templates.go]: Would it make more sense if this was called "bloattracker"?

**Comment by rminnich** [comment]
not sure what we want to do with this, get rid of it for now.
